test/shared/src/main/scala/zio/test/environment/TestClock.scala

```diff
@@ -386,6 +388,13 @@ object TestClock extends Serializable {
   def setTimeZone(zone: ZoneId): ZIO[TestClock, Nothing, Unit] =
     ZIO.accessM(_.get.setTimeZone(zone))
 
+  /**
+   * Accesses a `TestClock` instance in the environment and returns a new
+   * `Scheduler` backed by this `TestClock`.
+   */
+  val scheduler: ZIO[TestClock, Nothing, IScheduler] =
```

Comment:
This is not ideal. I tried to call the `Scheduler.scheduler` method from the package object that I created but then I got an error message that the `Scheduler` was not in the `TestEnvironment` even though I would have thought it would be since the `TestClock` now implements the `Scheduler`. Is there a better way to do this that I am just not understanding?

Reply:
Actually we can just make `TestClock.live` emit a scheduler too!

Reply:
To be clear, this would look like:

```scala
live : ZDep[Live, Nothing, Clock with TestClock with Scheduler]
```

The _old_ `Scheduler`, which was a type alias for `Has[IScheduler]`.